tests/Fixer/Phpdoc/PhpdocNoAliasTagFixerTest.php

```diff
@@ -25,39 +25,39 @@
 {
     public function testInvalidConfigCase1()
     {
-        $this->setExpectedException(
+        $this->setExpectedExceptionRegExp(
             'PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException',
-            '[phpdoc_no_alias_tag] Invalid configuration: Tag to replace must be a string.'
+            '#^\[phpdoc_no_alias_tag\] Invalid configuration: Tag to replace must be a string\.$#'
         );
 
         $this->fixer->configure(array(1 => 'abc'));
     }
 
     public function testInvalidConfigCase2()
     {
-        $this->setExpectedException(
+        $this->setExpectedExceptionRegExp(
             'PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException',
-            '[phpdoc_no_alias_tag] Invalid configuration: Tag to replace to from "a" must be a string.'
+            '#^\[phpdoc_no_alias_tag\] Invalid configuration: Tag to replace to from \"a\" must be a string\.$#'
         );
 
         $this->fixer->configure(array('a' => null));
     }
 
     public function testInvalidConfigCase3()
     {
-        $this->setExpectedException(
+        $this->setExpectedExceptionRegExp(
             'PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException',
-            '[phpdoc_no_alias_tag] Invalid configuration: Tag "see" cannot be replaced by invalid tag "link*/".'
+            '#^\[phpdoc_no_alias_tag\] Invalid configuration: Tag \"see\" cannot be replaced by invalid tag "link\*\/".$#'
         );
 
         $this->fixer->configure(array('see' => 'link*/'));
     }
 
     public function testInvalidConfigCase4()
     {
-        $this->setExpectedException(
+        $this->setExpectedExceptionRegExp(
             'PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException',
-            '[phpdoc_no_alias_tag] Invalid configuration: Cannot change tag "see" to tag "link", as the tag is set configured to be replaced to "see".'
+            '#^\[phpdoc_no_alias_tag\] Invalid configuration: Cannot change tag \"link\" to tag "see", as the tag is set configured to be replaced to \"link\"\.$#'
```

Comment:
My changes introduced a subtle difference in the way circular replacements are detected, I reverted it.